static/js/gtag.js

```diff
@@ -7,11 +7,22 @@ gtag('config', 'UA-26406144-13');
 // with a 'data-ga-click-event' attribute is clicked.
 window.addEventListener('DOMContentLoaded', function () {
   function sendEvent(e) {
-    gtag('event', e.currentTarget.dataset.gaClickEvent, {
+    var elem = e.currentTarget;
+
+    var data = {
       'event_category': 'click',
       'event_label': 'path:' + window.location.pathname,
       'value': 1
-    });
+    };
+
+    // external links should be
```

Comment:
Maybe expand this explanation a bit, and please link to the documentation for beacon: https://support.google.com/analytics/answer/7478520?hl=en

Reply:
ah, I forget to complete that sentence :)